openslides_backend/action/actions/mediafile/permission_mixin.py

```diff
@@ -0,0 +1,139 @@
+from typing import Any, Dict, List, Optional, Tuple
+
+from ....permissions.management_levels import OrganizationManagementLevel
+from ....permissions.permission_helper import has_organization_management_level
+from ....services.datastore.commands import GetManyRequest
+from ....shared.exceptions import (
+    ActionException,
+    MissingPermission,
+    PermissionException,
+)
+from ....shared.filters import And, Filter, FilterOperator, Not
+from ....shared.patterns import KEYSEPARATOR, Collection, FullQualifiedId
+from ...action import Action
+
+
+class MediafilePermissionMixin(Action):
```

Comment:
Thisis now more of a general mediafile mixin instead of a permission mixin, so please rename

Reply:
Renamed into `MediafileMixin`. I have also moved the mixin into a mixins file.